app/models/case.rb

```diff
@@ -150,8 +150,10 @@ def to_partial_path
     "cases/court_case"
   end
 
-  def formated_decision_date
-    self.decision_date.strftime("%B %d, %Y")
+  def formatted_decision_date
+    if !self.decision_date.blank?
```

Comment:
this is great! You can also do `self.decision_date.present?`

Reply:
That seems nicer, thanks! I'll change it.